bundles/org.openhab.ui.basic/src/main/java/org/openhab/ui/basic/internal/render/InputRenderer.java

```diff
@@ -45,6 +50,12 @@ public class InputRenderer extends AbstractWidgetRenderer {
 
     private final Logger logger = LoggerFactory.getLogger(InputRenderer.class);
 
+    static final Pattern NUMBER_PATTERN = Pattern.compile("^(\\+|-)?[0-9\\.,]+");
+    static final Pattern DOT_SEPARATOR_PATTERN = Pattern
+            .compile("^-?(([0-9]{1,3}(,[0-9]{3})*)|([0-9]*))?(\\.[0-9]+)?$");
+    static final Pattern COMMA_SEPARATOR_PATTERN = Pattern
+            .compile("^-?(([0-9]{1,3}(\\.[0-9]{3})*)|([0-9]*))?(,[0-9]+)?$");
```

Comment:
Please add "private"
Please also move the static before the non static

Reply:
Done.